src/test/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/services/PlanServiceTest.kt

```diff
@@ -27,7 +26,7 @@ class PlanServiceTest {
   private val planVersionRepository: PlanVersionRepository = mockk()
   private val planAgreementNoteRepository: PlanAgreementNoteRepository = mockk()
   private val planService = PlanService(planRepository, planVersionRepository, planAgreementNoteRepository)
-  private val plan: PlanEntity = mockk()
+  private val planEntity: PlanEntity = PlanEntity()
 
   @Nested
   @DisplayName("gePlayByUuid")
```

Comment:
We have to fix this.

Reply:
Yep is in my local branch for the PlanControllerTests, will fix in the PR for that - apparently we've had this for ages but I hadn't spotted it til today!